packages/account/src/Sections/Profile/FinancialAssessment/financial-information-list.js

```diff
@@ -297,3 +297,139 @@ export const account_turnover_list = [
         value: 'Over $500,000',
     },
 ];
+
+export const binary_options_trading_experience_list = [
+    {
+        text: localize('0-1 year'),
```

Comment:
Should we use the `<Localize ... />` component here instead?

Reply:
At the very least, the `binary_options_trading_experience_list` variable needs to be a function so it will have a chance at regenerating the localised string.

Reply:
will make it <Localize /> 👍